### ISSUE TITLE
Does NUnitlite support --Include?

### ISSUE THREAD

**Issue by lueleeqa** [nunit/nunit#3433]
https://github.com/nunit/docs/wiki/NUnitLite-Options  listed 

```
--include=CATEGORIES | Test CATEGORIES to be included. May be a single category, a comma-separated list of categories or a category expression.
```

Code:
```csharp
        public static void Main(string[] args)
        {
  
            new AutoRun().Execute(new string[] { "--include:Car Unit" });
            
        }
```

error:
NUnitLite 3.12.0 (.NET Framework 4.5)
Copyright (c) 2019 Charlie Poole, Rob Prouse

Invalid argument: --include:Car Unit

**Comment by CharliePoole** [comment]
That appears to be an out-of-date page. I notice that it isn't listed in the right-hand index of pages and doesn't seem to be linked from any other page. Most likely, that's why it has never been updated.

For a correct list of available options, run the test program with --help.

**Comment by mikkelbu** [comment]
This was closed by nunit/docs#540.
